src/openrct2/ride/Ride.h

```diff
@@ -907,8 +907,8 @@ constexpr uint8_t kMaxRideMeasurements = 8;
 constexpr money64 RIDE_VALUE_UNDEFINED = kMoney64Undefined;
 constexpr uint16_t kRideInitialReliability = ((100 << 8) | 0xFF); // Upper byte is percentage, lower byte is "decimal".
 
-#define STATION_DEPART_FLAG (1 << 7)
-#define STATION_DEPART_MASK (~STATION_DEPART_FLAG)
+constexpr uint8_t kStationDepartFlag = (1 << 7);
```

Comment:
`uint8_t` is exactly 8bits, which is what `1 << 7` needs, let's make sure everything is working correctly here,

Reply:
I am not sure about this one... Do you mean that `uint8_t` is not the correct type?

Reply:
I was unsure, just wanted to make sure a bit of testing was done around this. If `kStationDepartFlag` is ever taken as a signed integer, then it will lose the last bit.